backend/src/main/java/com/google/blackswan/mock/filesystem/LocalFileSystem.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.blackswan.mock.filesystem;
+
+import java.io.InputStream;
+import java.lang.ClassLoader;
+
+/** Local file system wrapper for data stored in resources/ folder. */
+public class LocalFileSystem implements FileSystem {
+  private final ClassLoader classLoader;
+
+  public static LocalFileSystem createSystem() {
+    return new LocalFileSystem();
+  }
```

Comment:
Same you can do return new LocalFileSystem(LocalFileSystem.class.getClassLoader())

Reply:
yep, fixed it. thank youu :)